UPGRADE-v3.md

```diff
@@ -145,4 +148,5 @@ Code BC changes
 - class `Tokens` is now `final`
 - method `create` of class `Config` has been removed, use the constructor
 - method `create` of class `RuleSet` has been removed, use the constructor
+- method `getClassyElements` of class `TokensAnalyzer` parameter `$returnTraitsImports` has been removed; now always returns trait import information
```

Comment:
it's internal class, so it's not BC break

Reply:
at least this, maybe
let's take this and merge the PR, but I'm happy for further discussion (and maybe even drop the notes about internal classes changes)

```suggestion

### BC breaks; common internal classes

- method `getClassyElements` of class `TokensAnalyzer` parameter `$returnTraitsImports` has been removed; now always returns trait import information
```

Reply:
Maybe when v3 is real close for release we can broom this file again and remove overcomplete details :+1: